homeworks/nikita.karchinskiy_Nikkarchinskiy/4-moduls/exports.js

```diff
@@ -0,0 +1,52 @@
+const monthIs = document.querySelector('[data-mounth-long-answer]');
```

Comment:
It would be best if you marked each DOM element variable with `El` postfix. For example, in your case, you need to name this variable as `monthIsEl`. Also, I don't understand what does it mean by `monthIs`.

1) Create a new name;
2) Add `el` postfix;
3) Review all your variables and add `el` postfix where it's needed.